server/config/index.js

```diff
@@ -32,6 +34,19 @@ function anyEnabled() {
 	} );
 }
 
+const ssrConfig = template(
+	'var data = <%= data %>;' +
+	'<%= config %>' +
+	'<%= isEnabled %>' +
+	'<%= anyEnabled %>'
+) ( {
+	data: JSON.stringify( clientData ),
+	config: config.toString(),
```

Comment:
I wouldn't worry about duplicating the config module on the client if I were you, I think we should avoid serializing functions here and depending on several globals.

1 global object containing only the raw configuration should be enough for our needs:
```
window.configData = <%= data %>;
```

Moreover, we could easily share the config module between the client and the server.
```
function loadConfig( configData = null ) {
	data = configData || window && window.configData;
}
```